e2e/specs/st_tabs.spec.js

```diff
@@ -36,11 +36,18 @@ describe("st.tabs", () => {
       2
     );
 
-    cy.get("[data-testid='stSidebar'] .stTabs")
+    cy.get("[data-testid='stSidebar'] .stTabs .st-cx")
```

Comment:
I'm not sure if we can or should use `.st-cx` since this is probably an internally generated class. I think it is fine to just select the first `.stMarkdown` within the `.stTabs` component: 

```javascript
cy.get("[data-testid='stSidebar'] .stTabs")
.first()
      .within(() => {
        cy.get(".stMarkdown").first().should("have.text", "I am in the sidebar");
      });
```